### PR TITLE
release: Bump version to 6.0.0 [skip ci]

### PR DESCRIPTION
**Pull request by matsales28**
@VSPPedro the bumping PR is ready. Once we have the changelog I can regenerate the docs, publish to RubyGems and create the new tag on GitHub.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 6.0.0 - 2022-12-22
```
Comment (VSPPedro) [inline CHANGELOG.md:3]: Oops 2023 😅

Comment (matsales28) [reply CHANGELOG.md:3]: Oops, that's why we only merge with reviews. I'll update it, there's also some minor things in the `README.md` still referencing version 5, I'll create a PR for updating this.

Comment (VSPPedro) [reply CHANGELOG.md:3]: No problem, @matsales28,  and congratulations for the release! Nice job!

Comment (matsales28) [reply CHANGELOG.md:3]: Actually, do we need a new version for things like that?

**Pull request merged**